deploy/openshift4/nsx-ipi-helper.yaml

```diff
@@ -0,0 +1,51 @@
+# nsx-ipi-helper DaemonSet
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nsx-ipi-helper
+  namespace: nsx-system-operator
+  labels:
+    tier: nsx-networking
+    component: nsx-ipi-helper
+    version: v1
+spec:
+  selector:
+    matchLabels:
+      tier: nsx-networking
+      component: nsx-ipi-helper
+      version: v1
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        tier: nsx-networking
+        component: nsx-ipi-helper
+        version: v1
+    spec:
+      hostNetwork: true
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+        - key: node.kubernetes.io/not-ready
+          effect: NoSchedule
+        - key: node.kubernetes.io/unreachable
+          effect: NoSchedule
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      initContainers:
+        - name: nsx-ipi-helper
+          image: nsx-ujo-docker-local.artifactory.eng.vmware.com/3.1.1.17415517/nsx-ncp-rhel:latest
```

Comment:
It's a temp image for verification.

Reply:
Didn't find an offical image including both curl and iptables so I'm using ubi8 and exec "yum -y install iptables" in the command...